css-checks/src/main/java/org/sonar/css/checks/common/FontFamilyNotEndingWithGenericFontFamilyCheck.java

```diff
@@ -85,18 +108,24 @@ private boolean isPotentialGenericFamily(@Nullable Tree tree) {
       return false;
     }
 
-    if (!tree.is(Tree.Kind.IDENTIFIER, Tree.Kind.LESS_VARIABLE, Tree.Kind.SCSS_VARIABLE, Tree.Kind.VARIABLE, Tree.Kind.FUNCTION)) {
+    if (!tree.is(Tree.Kind.IDENTIFIER, Tree.Kind.STRING, Tree.Kind.LESS_VARIABLE, Tree.Kind.SCSS_VARIABLE, Tree.Kind.VARIABLE, Tree.Kind.FUNCTION)) {
       return false;
     }
 
     if (tree.is(Tree.Kind.IDENTIFIER)) {
       IdentifierTree identifier = (IdentifierTree) tree;
       if (!identifier.isInterpolated()
         && !FontFamily.GENERIC_FAMILY_NAMES.contains(identifier.text().toLowerCase())
-        && !StandardProperty.COMMON_VALUES.contains(identifier.text().toLowerCase())) {
+        && !StandardProperty.COMMON_VALUES.contains(identifier.text().toLowerCase())
+        && !identifier.text().matches(exclusions)) {
         return false;
       }
     }
+
+    if (tree.is(Tree.Kind.STRING) && !((StringTree) tree).actualText().matches(exclusions)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)